tensorly/backend/mxnet_backend.py

```diff
@@ -207,6 +218,17 @@ def partial_svd(matrix, n_eigenvecs=None):
         U, S, V = U[:, ::-1], S[::-1], V[:, ::-1]
         return tensor(U, **ctx), tensor(S, **ctx), tensor(V.T, **ctx)
 
+def qr(matrix, **kwds):
+    # NOTE - should be replaced with geqrf when available
+    try:
+        Q, R = map(nd.transpose, nd.linalg.gelqf(matrix.T))
+    except AttributeError:
+        warnings.warn('This version of MXNet does not include the linear '
+                      'algebra function gelqf(). Substituting with numpy.')
+        Q, R = map(tensor, numpy_backend.qr(to_numpy(matrix), **kwds))
```

Comment:
Similarly, I would just write 
```
Q, R = numpy_backend.qr(to_numpy(matrix))
return tensor(Q), tensor(R)
```
I'd also skip the **kwds here.

Reply:
How come? I suppose the `qr()` keywords differ between backends...

Reply:
Since i) it is only used in the NumPy case so leads to potentially inconsistent behaviour and ii) it's not really needed :)

Reply:
Sure.